### PR TITLE
Add `pathvars` & `body` in the store

### PR DESCRIPTION
**Pull request by xurei**
It would be useful to have the path vars and/or the body of the request in the store. I personnaly use those information to detect if the data needs to be fetched again.

Here is my proposal for such feature.

### CODE REVIEW

**Comment by lexich** [comment]
Hi @xurei thx for you PR. I need time to test you solution, but I like your proposal. 👍

**Comment by lexich** [comment]
@xurei you code in 0.10.7
Good job

**Comment by xurei** [comment]
Thanks :-)

Just thought about something though. If you bind multiple action to the same store, that might create some weird behavior... Any idea how we could improve the implementation for such cases ?

BTW, I made a react component based on your package : [redux-api-react-switch](https://github.com/xurei/redux-api-react-switch)
Any feedback about it ?

**Comment by lexich** [comment]
@xurei Can you write example of weird behaviour?

redux-api-react-switch is very interesting project 👍

**Comment by xurei** [comment]
I'll create an issue with the details ;-)

File: src/reducerFn.js [hunk 1]
```diff
@@ -14,21 +14,20 @@ export default function reducerFn(initialState, actions={}, reducer) {
   const { actionFetch, actionSuccess, actionFail,
     actionReset, actionCache, actionAbort } = actions;
   return (state=initialState, action)=> {
+    const params = action.params || {};
```
Comment (eps1lon) [inline src/reducerFn.js]: Params are stored in action.request.params. This is currently always defaulting to {}

File: src/reducerFn.js [hunk 2]
```diff
@@ -14,21 +14,20 @@ export default function reducerFn(initialState, actions={}, reducer) {
...
     switch (action.type) {
       case actionFetch:
         return {
           ...state,
-          pathvars:{},
-          body:{},
+          pathvars: action.pathvars || {},
```
Comment (eps1lon) [inline src/reducerFn.js]: Same here. Pathvars are stored in action.request.pathvars. This is currently defaulting.

**Comment by lexich** [comment]
Hi @eps1lon I'll be glad to see new PR with bug fixing 👍

**Pull request merged**